app/src/main/java/org/mozilla/focus/browser/BrowserFragment.kt

```diff
@@ -173,6 +173,13 @@ class BrowserFragment : IWebViewLifecycleFragment() {
         return layout
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```

Comment:
I assume that Pocket Videos will just be loading a fragment anyway. This breaks history behavior if users expect "pocket to be a web page" and want to go "back" to it, but since back doesn't work on Youtube anyway (#733), I didn't handle that case here.